schematic/visualization/attributes_explorer.py

```diff
@@ -19,31 +18,38 @@
 class AttributesExplorer:
     """AttributesExplorer class"""
 
+    # pylint: disable=R0913
```

Comment:
@andrewelamb @mialy-defelice can I add this line to bypass the complaint of "too many arguments": 
```
schematic/visualization/attributes_explorer.py:21:4: R0913: Too many arguments (6/5) (too-many-arguments)
```

I tried changing the class to `dataclass` but still I got: 
```
schematic/visualization/attributes_explorer.py:19:0: R0902: Too many instance attributes (11/7) (too-many-instance-attributes)
```

If this needs to be addressed, can we address it in a separate PR?

Reply:
I think that's ok to do. But to make it more clear I would do  #pylint: disable=too-many-arguments

Reply:
Yes I don't think dataclass is appropriate here. I would add `#pylint: disable=too-many-instance-attributes` for now.

Reply:
@andrewelamb Thanks for the feedback. I will update this part based on what you advised. But since I am not using dataclass, I assume that adding `#pylint: disable=too-many-arguments` will be sufficient?

Reply:
Yep!